src/sentry/dynamic_sampling/rules/base.py

```diff
@@ -0,0 +1,58 @@
+from typing import List, OrderedDict, Set
+
+import sentry_sdk
+
+from sentry import quotas
+from sentry.dynamic_sampling.rules.biases.base import Bias, BiasParams
+from sentry.dynamic_sampling.rules.combine import get_relay_biases_combinator
+from sentry.dynamic_sampling.rules.logging import log_rules
+from sentry.dynamic_sampling.rules.utils import BaseRule, RuleType, get_enabled_user_biases
+from sentry.models import Project
+
+ALWAYS_ALLOWED_RULE_TYPES = {RuleType.UNIFORM_RULE}
+
+
+def _get_guarded_blended_sample_rate(project: Project) -> float:
```

Comment:
i don't recall why it has `_` in the name.


Single underscore at the beginning:

Python doesn't have real private methods. Instead, one underscore at the start of a **method** or attribute name means you shouldn't access this method, because it's not part of the API.

So in this case you can just rename to `get_guarded_blended_sample_rate `